### PR TITLE
✨ adds infonline anonymous analytics to amp

### PR DESCRIPTION
**Pull request by infonline**
This will add an anonymous variant of the INFOnline analytics to the AMP project.

### CODE REVIEW

**Comment by infonline** [comment]
Are there any updates on this PR?

**Comment by avimehta** [comment]
@zhouyx to review.

File: examples/analytics-vendors.amp.html [hunk 1]
```diff
@@ -715,6 +716,30 @@
 </amp-analytics>
 <!-- End INFOnline example -->
 
+<!--
```
Comment (zhouyx) [inline examples/analytics-vendors.amp.html:719]: Hello @infonline. Can I ask why we need the infonline_anonymous as a different vendor? 
Is it possible to set to anonymous mode via a var? For example
```
<amp-analytics type=infonline>
<script type="application/json">
{
  "vars": {
     "anonymous": true
   }
   ...
}
</script>
</amp-analytics>
```

Comment (infonline) [reply examples/analytics-vendors.amp.html:719]: Thank you very much for reviewing the PR. Due to privacy protection requirements, 
we want to completely separate the previous solution from the new anonymous solution. 
We also want to avoid technical dependencies at this point.

File: extensions/amp-analytics/0.1/vendors.js [hunk 2]
```diff
@@ -54,6 +54,7 @@ import {GOOGLEADWORDS_CONFIG} from './vendors/googleadwords';
 import {GOOGLEANALYTICS_CONFIG} from './vendors/googleanalytics';
 import {GTAG_CONFIG} from './vendors/gtag';
 import {IBEATANALYTICS_CONFIG} from './vendors/ibeatanalytics';
+import {INFONLINE_ANONYMOUS_CONFIG} from './vendors/infonline_anonymous';
```
Comment (zhouyx) [inline extensions/amp-analytics/0.1/vendors.js:57]: We recent refactored this file, you no longer need to make any change to this file : )

File: extensions/amp-analytics/0.1/vendors/infonline_anonymous.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  },
+  'triggers': {
+    'pageview': {
+      'on': 'visible',
+      'request': 'pageview',
+    },
+  },
+});
+
+export {INFONLINE_ANONYMOUS_CONFIG};
```
Comment (zhouyx) [inline extensions/amp-analytics/0.1/vendors/infonline_anonymous.js:61]: same: This file is no longer needed. Now we rely on the config from the .json  file

File: extensions/amp-analytics/0.1/vendors/infonline_anonymous.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  "vars": {
+    "ap": "1"
+  },
+  "transport": {
+    "beacon": false,
+    "xhrpost": false,
+    "image": true
+  },
+  "requests": {
+    "pageview": "${url}?ap=${ap}&dn=${dn}&cn=${cn}&st=${st}&co=${co}&cp=${cp}&au=${canonicalUrl}&ar=${documentReferrer}&ash=${screenHeight}&asw=${screenWidth}&tp=pageview",
```
Comment (zhouyx) [inline extensions/amp-analytics/0.1/vendors/infonline_anonymous.json:11]: FYI: since we can't encode the `${url}` value, please make sure that your client specify it in the `requests` setting, not the `vars` setting. 
I think you've known this already : )

File: examples/analytics-vendors.amp.html (hunk 1 above)
Comment (zhouyx) [reply examples/analytics-vendors.amp.html:719]: SGTM. We want to best support your use case. Given all vendor configs are loaded separately and lazily, we are good with having the anonymous solution registered as a standalone vendor.

**Comment by zhouyx** [comment]
LGTM

File: EXTENSIONS_CSS_MAP [hunk 5]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (zhouyx) [inline EXTENSIONS_CSS_MAP:1]: @infonline could you revert the change here? Thanks!

Comment (infonline) [reply EXTENSIONS_CSS_MAP]: done

**Comment by zhouyx** [comment]
@rcebulko wanna ask about the owner check to the `EXTENSIONS_CSS_MAP` file. 
image

It's indicated that there's no change to that file. (Though I have no idea why GH is even showing it in the files changed) . Any idea why owner check is requested? 

Also to @dvoytenko for approval as suggested by the owner check bot.

**Comment by rcebulko** [comment]
> @rcebulko wanna ask about the owner check to the `EXTENSIONS_CSS_MAP` file.
> image
> 
> It's indicated that there's no change to that file. (Though I have no idea why GH is even showing it in the files changed) . Any idea why owner check is requested?
> 
> Also to `@dvoytenko` for approval as suggested by the owner check bot.

It looks like this PR is adding a new, empty file in the root directory. Is that intentional?

**Comment by zhouyx** [comment]
oops my bad! Didn't realize that's a new file. It makes total sense now. Thanks

**Comment by zhouyx** [comment]
@infonline could you please revert the empty EXTENSIONS_CSS_MAP  file in the root directory? Looks like an unexpected change. Thanks

**Comment by infonline** [comment]
I don't know where the file came from. I removed it from root. Thanks.

**Comment by zhouyx** [comment]
Thank you @infonline. Pull Request merged.
